### ISSUE TITLE
Text files are Windows-specific

### ISSUE THREAD

**Issue by blueberry** [gpu/JOCLBlast#4]
This might not be a real issue, but I am reporting it anyway. When opened in emacs on linux, JOCLBlast Java source contains of ^M characters.

Like in this stackexchange question:
http://unix.stackexchange.com/questions/17647/m-at-the-end-of-each-line-of-text-files-generated-under-windows

**Comment by gpu** [comment]
Yes, these line endings.... Either Visual Studio complains about "wrong" line endings, or the Linux editors show odd characters. There is a dedicated article, https://help.github.com/articles/dealing-with-line-endings/ , showing how to "clean up" the line endings, but it seems a bit brute force - and it's a bit more difficult in this case, as most of the code in JOCLBlast is auto-generated, and I'm not sure whether it's so easy to sneak a different `line.separator` into the print writers on Windows... 

I'll leave this open for now. It does not have the highest priority, but I also don't like when files appear to be completely "modified" only because of the line endings, and a having a generic solution for this would indeed be nice.
